docs/_docs/framework/capabilities/manage/finops-assessment.md

```diff
@@ -0,0 +1,35 @@
+<a name="definition"></a>
+**FinOps Assessment refers to assessing FinOps practice and measure its effectiveness, align activities with organizational goals, and identify areas for improvement, with the aim of becoming more mature.**
+{: .fs-6 .fw-300 }
+
+FinOps Capabilities can be evaluated at any point in an organization’s FinOps journey.
+The objective of the FinOps assessment is to capture the maturity of a set of activities, define the objectives according to the organization, and outline the path to achieve that goal. Setting KPIs will help the FinOps team identify which objectives are being met and which areas need more attention to increase maturity.
+When conducting an assessment, organizations can evaluate their current capabilities and reassess the capabilities that will have the most impact on maximizing cloud value. These assessments are carried out by a target group consisting of a group composed of different personas that bring different points of view on what is essential. As organizations gain maturity, they will implement more capabilities.
+
+<br>
+
+
+## ✋ Before you begin
+
+For organizations that are just starting out, they can conduct the Microsoft FinOps Review. 
+Leverage from FinOps guidance to assess your organization’s capability gaps. Start small, choose one or two capabilities to focus and that are a priority to maximize the cloud value. Finishing the assessment, you will get recommendations for maximizing cloud business value using FinOps best practices.
+Additionally, document all assessments to be available for the FinOps team and define annual goals with clear KPI’s based on the recommendations 
+
+
+<br>
+
+
+Getting started
+Find out what capabilities are relevant for the FinOps assessment, and confirm if KPIs are being met. The output of the FinOps assessment is used to monitor and evaluate annual objectives. If needed, adjust goals after each assessment.
+Additionally, build a consistent format that enables simpler comparisons between Target Groups and over time
+
+
+<br>
+
+## 🏗️ Building on the basics
+
+As the organization gains more maturity in its capabilities, more capabilities will be assessed, based on FinOps KPIs. Each Business Unit leads its own FinOps Assessment at a regular cadence.
+
+## ⏩ Next steps
+
+<br>
```

Comment:
Should we include the "Learn more at the FinOps Foundation" section? 

## 🍎 Learn more at the FinOps Foundation

This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Assessment capability](https://www.finops.org/framework/capabilities/finops-assessment/) article in the FinOps Framework documentation.